test/coverage-report.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+if [ $# -lt 3 ]; then
```

Comment:
The fourth argument is "optional" since we wouldn't have pr args for patch builds and other local changes, so it can be an empty string that is not detected by the script. I've added another check to make sure that we are either getting 3 or 4 arguments to make sure the script fails straight away if the incorrect number of arguments are given. see [f6c41f6](https://github.com/wiredtiger/wiredtiger/pull/10915/commits/f6c41f665c9227651419aa3517491c20f8e1f7f3)